docs/CONFIG.md

```diff
@@ -2517,7 +2517,7 @@ GasLimit = 5400000
 AutoCreateKey = true
 BlockBackfillDepth = 10
 BlockBackfillSkip = false
-FinalityDepth = 50
+FinalityDepth = 10
```

Comment:
why are we reducing the BSC finality depth to 10?

Reply:
And same question for the chains below

Reply:
Updated.